Snippets/AppContext/01-app-context.swift

```diff
@@ -0,0 +1,272 @@
+//
+//  01-app-context.swift
+//
+//  Copyright (c) PubNub Inc.
+//  All rights reserved.
+//
+//  This source code is licensed under the license found in the
+//  LICENSE file in the root directory of this source tree.
+//
+
+// snippet.import
+import PubNubSDK
+
+// snippet.end
+
+// snippet.pubnub
+// Initializes a PubNub object with the configuration
+let pubnub = PubNub(
+  configuration: PubNubConfiguration(
+    publishKey: "demo",
+    subscribeKey: "demo",
+    userId: "myUniqueUserId"
+  )
+)
+
+// snippet.end
+
+// snippet.all-user-metadata
+// Retrieve all user metadata
+pubnub.allUserMetadata { result in
+  switch result {
+  case let .success(response):
+    print("The user metadata objects: \(response.users)")
+    print("The next page used for pagination: \(String(describing: response.next))")
+  case let .failure(error):
+    print("Fetch All request failed with error: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.fetch-user-metadata
+pubnub.fetchUserMetadata("some-id") { result in
+  switch result {
+  case let .success(userMetadata):
+    print("The metadata for `\(userMetadata.metadataId)`: \(userMetadata)")
+  case let .failure(error):
+    print("Fetch request failed with error: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.set-user-metadata
+let userMetadataToSet = PubNubUserMetadataBase(
+  metadataId: "some-id",
+  name: "Some User",
+  custom: ["department": "Engineering"]
+)
+
+pubnub.setUserMetadata(userMetadataToSet) { result in
+  switch result {
+  case let .success(userMetadata):
+    print("The metadata for `\(userMetadata.metadataId)`: \(userMetadata)")
+  case let .failure(error):
+    print("Create request failed with error: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.remove-user-metadata
+pubnub.removeUserMetadata("some-id") { result in
```

Comment:
Is there any requests to have snippet without userId which will use ID from PubNub client itself?